### DISCUSSION TITLE
How to get real client IPs in query logs instead of docker bridge IP

### DISCUSSION THREAD

**Discussion by kzshantonu** [AdguardTeam/AdGuardHome#4101] [Q&A]
Noob question probably but any help will be appreciated

**Comment by ainar-g** [comment]
Hello.  The most direct solution would be to use Docker's `--network host` option when starting your container, which disables the isolation of the host network stack from the guest machine.

**Reply by kzshantonu** [reply]
thanks. what's the correct way to add this to compose yml?

**Reply by bh** [reply]
> thanks. what's the correct way to add this to compose yml?
https://www.google.com/search?q=docker+compose+network+host

**Reply by kzshantonu** [reply]
> > thanks. what's the correct way to add this to compose yml?
> https://www.google.com/search?q=docker+compose+network+host
> 
> 

Done that. I've seen conflicting information. Some say to do it like: 

```
networks:
  hosts:
```

Others say:

`network_mode: hosts`
